alacritty/src/event.rs

```diff
@@ -923,6 +932,29 @@ impl<N: Notify + OnResize> Processor<N> {
                     },
                     TerminalEvent::MouseCursorDirty => processor.reset_mouse_cursor(),
                     TerminalEvent::Exit => (),
+                    TerminalEvent::CursorStartBlinking => {
+                        let rate = processor.ctx.config.cursor.blink_rate;
+                        let scheduler = &mut processor.ctx.scheduler;
+                        if rate != 0 && scheduler.get_mut(TimerId::CursorBlinking).is_none() {
```

Comment:
What if the configuration file is changed to disable blinking while Alacritty is active? We should support live config reload for this.